docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -618,3 +618,125 @@ DROP TABLE IF EXISTS SITEASSOC_CONTEXT_ASSOCIATIO;
+    PRIMARY KEY(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_ASSIGNMENT_ATTACHMENTS (
+    ASSIGNMENT_ID VARCHAR(36) NOT NULL,
+    ATTACHMENT    VARCHAR(1024),
+    CONSTRAINT FK_HYK73OCKI8GWVM3AJF8LS08AC FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID),
+    INDEX FK_HYK73OCKI8GWVM3AJF8LS08AC(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_ASSIGNMENT_GROUPS (
+    ASSIGNMENT_ID VARCHAR(36) NOT NULL,
+    GROUP_ID      VARCHAR(255),
+    CONSTRAINT FK_8EWBXSPLKE3C487H0TJUJVTM FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID),
+    INDEX FK_8EWBXSPLKE3C487H0TJUJVTM(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_ASSIGNMENT_PROPERTIES (
+    ASSIGNMENT_ID VARCHAR(36) NOT NULL,
+    VALUE         LONGTEXT,
+    NAME          VARCHAR(255) NOT NULL,
+    PRIMARY KEY(ASSIGNMENT_ID,NAME),
+    CONSTRAINT FK_GDAT1B6UQIUI9MXDKTD6M5IG1 FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_SUBMISSION (
+    SUBMISSION_ID    VARCHAR(36) NOT NULL,
+    CREATED_DATE     DATETIME,
+    MODIFIED_DATE    DATETIME,
+    RETURNED_DATE    DATETIME,
+    SUBMITTED_DATE   DATETIME,
+    FACTOR           INT,
+    FEEDBACK_COMMENT LONGTEXT,
+    FEEDBACK_TEXT    LONGTEXT,
+    GRADE            VARCHAR(32),
+    GRADE_RELEASED   BIT,
+    GRADED           BIT,
+    GRADED_BY        VARCHAR(36),
+    GROUP_ID         VARCHAR(36),
+    HIDDEN_DUE_DATE  BIT,
+    HONOR_PLEDGE     BIT,
+    RETURNED         BIT,
+    SUBMITTED        BIT,
+    TEXT             LONGTEXT,
+    USER_SUBMISSION  BIT,
+    ASSIGNMENT_ID    VARCHAR(36),
+    PRIMARY KEY(SUBMISSION_ID),
+    CONSTRAINT FK_6A25A0BXIFPYEIJ72PDK7XRLR FOREIGN KEY(ASSIGNMENT_ID) REFERENCES ASN_ASSIGNMENT(ASSIGNMENT_ID),
+    INDEX FK_6A25A0BXIFPYEIJ72PDK7XRLR(ASSIGNMENT_ID)
+);
+
+CREATE TABLE ASN_SUBMISSION_ATTACHMENTS (
+    SUBMISSION_ID VARCHAR(36) NOT NULL,
+    ATTACHMENT    VARCHAR(1024),
+    CONSTRAINT FK_JG017QXC4PV3MDF07C1XPYTB8 FOREIGN KEY(SUBMISSION_ID) REFERENCES ASN_SUBMISSION(SUBMISSION_ID),
+    INDEX FK_JG017QXC4PV3MDF07C1XPYTB8(SUBMISSION_ID)
+);
+
+CREATE TABLE ASN_SUBMISSION_FEEDBACK_ATTACH (
+    SUBMISSION_ID       VARCHAR(36) NOT NULL,
+    FEEDBACK_ATTACHMENT VARCHAR(1024),
+    CONSTRAINT FK_3DOU5GSQCYA4RWWY99L91FOFB FOREIGN KEY(SUBMISSION_ID) REFERENCES ASN_SUBMISSION(SUBMISSION_ID),
+    INDEX FK_3DOU5GSQCYA4RWWY99L91FOFB(SUBMISSION_ID)
+);
+
+CREATE TABLE ASN_SUBMISSION_PROPERTIES (
+    SUBMISSION_ID VARCHAR(36) NOT NULL,
+    VALUE         LONGTEXT,
+    NAME          VARCHAR(255) NOT NULL,
+    PRIMARY KEY(SUBMISSION_ID,NAME),
+    CONSTRAINT FK_2K0JAT40WAP5EKWKPSN201EAU FOREIGN KEY(SUBMISSION_ID) REFERENCES ASN_SUBMISSION(SUBMISSION_ID)
+);
+
+CREATE TABLE ASN_SUBMISSION_SUBMITTER (
+    ID            BIGINT NOT NULL AUTO_INCREMENT,
+    FEEDBACK      LONGTEXT,
+    GRADE         VARCHAR(255),
+    SUBMITTEE     BIT NOT NULL,
+    SUBMITTER     VARCHAR(36) NOT NULL,
+    SUBMISSION_ID VARCHAR(36) NOT NULL,
```

Comment:
these should match sakai_user_id_map user_id field length

Reply:
SUBMISSION_ID is correct it can't be created by users